### DISCUSSION TITLE
I want to use 'STM32F103C8' without crystal.

### DISCUSSION THREAD

**Discussion by SeongJongKwak** [stm32duino/Arduino_Core_STM32#2376] [General]
I want to use 'STM32F103C8' without crystal. 

hello. Nice to meet you. What I'm curious about is this: 

1. I want to make 'STM32F103C8' without 'crystal'. 

2. Is it okay to just remove the ‘crystal’ from the circuit?

3. Can I use the boot file ‘generic_boot20_pc13’ as is?

4. Can firmware be done in ‘Arduino IDE’?

If anyone knows please help.

**Comment by fpistm** [comment]
Please avoid to open several entry for the same questions:
https://www.stm32duino.com/viewtopic.php?p=14261&sid=424c2af62a337d38a001ac8b16150b3a#p14261
